.github/workflows/tools.yml

```diff
@@ -174,14 +174,17 @@ jobs:
               cat temp-output
               tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
               rm temp-output
-          - id: libuv
-            subsystem: deps
-            label: dependencies
-            run: |
-              ./tools/dep_updaters/update-libuv.sh > temp-output
-              cat temp-output
-              tail -n1 temp-output | grep "NEW_VERSION=" >> "$GITHUB_ENV" || true
-              rm temp-output
+          # libuv update was disabled because of Feb 14 security release modified
```

Comment:
Maybe add the year for context and the version where it happened? This looks like a temporary solution, meaning it could last a couple of months or a few years and better provide the entire context for people checking  this in the future?

Reply:
Shouldn't be it referenced by the commit history?